app/uk/gov/hmrc/incorporatedentityidentification/connectors/RegisterWithMultipleIdentifiersConnector.scala

```diff
@@ -81,7 +81,7 @@ object RegisterWithMultipleIdentifiersHttpParser {
             safeId <- (response.json \ IdentificationKey \ 0 \ IdentificationValueKey).validate[String]
           } yield safeId) match {
             case JsSuccess(safeId, _) => RegisterWithMultipleIdentifiersSuccess(safeId)
-            case _: JsError => throw new InternalServerException(s"Invalid JSON returned on Register API: ${response.body}")
+            case _ => RegisterWithMultipleIdentifiersFailure(response.status, response.body)
```

Comment:
This was correct as it was before, we don't expect to have invalid json sent to us so if it does happen we should throw an exception